src/System.Data.OleDb/tests/OleDbDataAdapterTests.cs

```diff
@@ -144,7 +144,7 @@ public void Fill_OpenDataReader_Throws()
                         }
                         else
                         {
-                            Assert.NotNull(adapter.Fill(ds, tableName));
+                            Assert.True(adapter.Fill(ds, tableName) >= 0);
```

Comment:
Nit: Can you make this Assert.InRange instead, e.g.
```C#
Assert.InRange(adapter.Fill(ds, tableName), 0, int.MaxValue);
```

Reply:
Thank you for the catch!